modules/redpanda/redpanda_test.go

```diff
@@ -278,6 +293,117 @@ func TestRedpandaWithTLS(t *testing.T) {
 	require.Error(t, results.FirstErr(), kerr.UnknownTopicOrPartition)
 }
 
+func TestRedpandaListener_Simple(t *testing.T) {
+	ctx := context.Background()
+
+  	// 1. Create network 
+	RPNetwork, err := network.New(ctx, network.WithCheckDuplicate())
```

Comment:
Nit: let's use lowercase here. Please apply it everywhere
```suggestion
	rpNetwork, err := network.New(ctx, network.WithCheckDuplicate())
```